projects/plutus/test/test_calibration.c

```diff
@@ -0,0 +1,64 @@
+#include <inttypes.h>
```

Comment:
Please use consistent naming between your test and your module

Reply:
Done